_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,336 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [text node][] that is [visible][] and a [child][] in the [flat tree][] of an HTML element, whose content changes through an action that was not initiated by the user, anytime after the [readiness][document readiness] of the document the [text node][] belongs to is equal to "complete".
```

Comment:
Can we rephrase the applicability, it is a bit hard to read & comprehend.

Reply:
When would be a case of a text node that is not a child of HTML element? Are you trying to exclude SVG text nodes?

Reply:
I tried to split it into items so that it is, hopefully, easier to read.
I've updated the applicability to consider any text node in the flat tree, so SVG text nodes are considered.